### DISCUSSION TITLE
Round Slider replacement

### DISCUSSION THREAD

**Discussion by wbagdon** [home-assistant/frontend#8161] [Dashboard]
Per @bramkragten  
> We are actually thinking about dropping the round slider in our cards, so we will not merge this until we made a decision about that

This is blocking #7803 and I don't see any discussion or issue about this feature.
I am opening this discussion to discuss the round slider.

**Comment by zsarnett** [comment]
We should just use buttons in a better way. Sliders are not user friendly on Mobile and definitely not easy to be mobile responsive
